pymeasure/instruments/instrument.py

```diff
@@ -126,50 +141,70 @@ class Instrument:
 
     # Variable holding the list of DynamicProperty parameters that are configurable
     # by users
-    _fget_params_list = ('get_command',
-                         'values',
-                         'map_values',
-                         'get_process',
-                         'command_process',
-                         'check_get_errors')
-
-    _fset_params_list = ('set_command',
-                         'validator',
-                         'values',
-                         'map_values',
-                         'set_process',
-                         'command_process',
-                         'check_set_errors')
+    _fget_params_list = (
+        "get_command",
+        "values",
+        "map_values",
+        "get_process",
+        "command_process",
+    )
+
+    _fset_params_list = (
+        "set_command",
+        "validator",
+        "values",
+        "map_values",
+        "set_process",
+        "command_process",
```

Comment:
Removed all `check_errors` parameters because we now check errors each time we communicate with the instrument.